### ISSUE TITLE
Missing support for cylinder collision shapes.

### ISSUE THREAD

**Issue by EzraT** [godotengine/godot-blender-exporter#300]
**OS:** Linux Mint 19.2

**Godot version:** 3.1.1

**Blender version:** 2.8 official

**Issue description:**

Godot versions 3.1.x and up support cylinder collision shapes, the blender exporter still misses this feature.

![01](https://user-images.githubusercontent.com/781121/70667299-cf6a8180-1c70-11ea-9d40-16faffa4636d.jpg)

Could this be added?

Thanks in advance.

**Comment by EzraT** [comment]
I decided to give this a shot myself, and I think I have it close to working, but the rotation of the CollisionShape is incorrect and I don't really know how to correct it.
First time working on a blender plugin so I'm not sure if I'm doing this right.

![14:21:36-2019-12-23](https://user-images.githubusercontent.com/781121/71360286-9a72fe80-258f-11ea-9e51-4b7bd8bcc625.png)

Here's what I have:
```
elif rbd.collision_shape == "CYLINDER":
            col_shape = InternalResource("CylinderShape", col_name)
            col_shape['radius'] = max(bounds.x, bounds.y) / 2
            col_shape['height'] = bounds.z
            shape_id = escn_file.add_internal_resource(col_shape, rbd)
```

Anybody got any ideas on how to correct the rotation?

**Comment by Calinou** [comment]
In Godot, by default, capsule and cylinder collision shapes are horizontal, so you need to offset their rotation by 90 degrees on the X axis (if I'm not mistaken).

We'll probably change this for 4.0 to make them upright by default.

**Comment by EzraT** [comment]
> In Godot, by default, capsule and cylinder collision shapes are horizontal

Aren't cylinder shapes vertical by default? I thought the capsule shape was the only one that was horizontal by default.

**Comment by Calinou** [comment]
@EzraT You're correct, only capsule shapes are rotated by default. I thought both were rotated for consistency, but well.

**Comment by EzraT** [comment]
> you need to offset their rotation by 90 degrees on the X axis to make them upright

@Calinou I dont think I can do this looking at the way the plugin works right now, I would mess up the rotation of the Imported MeshInstance, since its parented to the CollisionShape node when loaded into godot.
Is there a specific reason why its setup like this? 
Wouldn't it be a better idea to parent the MeshInstance to the Static/Rigidbody node instead?
This way the Shape node can be offset easier without changing the rotation of the Mesh.

**Comment by pwab** [comment]
>Wouldn't it be a better idea to parent the MeshInstance to the Static/Rigidbody node instead?

I totally agree with this. I don't know what should be the purpose of a CollisionShape with a Mesh as a child - never used it this way. From my point of view a RigidBody has a mesh and a shape - so both should be children of the RigidBody.
